components/chart/types/rangecolumn.md

```diff
@@ -0,0 +1,204 @@
+                     Type="ChartSeriesType.RangeColumn"
+                     FromField="@nameof(ScoreModel.LowScore)"
+                     ToField="@nameof(ScoreModel.HighScore)"
+                     CategoryField="@nameof(ScoreModel.Year)">
+        </ChartSeries>
+        <ChartSeries Name="University 2"
+                     Data="@StudentScoreList2"
+                     Type="ChartSeriesType.RangeColumn"
+                     FromField="@nameof(ScoreModel.LowScore)"
+                     ToField="@nameof(ScoreModel.HighScore)"
+                     CategoryField="@nameof(ScoreModel.Year)">
+        </ChartSeries>
+    </ChartSeriesItems>
+
+    <ChartTooltip Visible="true">
+        <Template>
+            @{
+                var item = (ScoreModel)context.DataItem;
+                <span>@item.LowScore - @item.HighScore points</span>
+            }
+        </Template>
+    </ChartTooltip>
+
+    <ChartTitle Text="Exam Score Ranges"></ChartTitle>
+
+    <ChartLegend Position="ChartLegendPosition.Right"></ChartLegend>
+</TelerikChart>
+
+@code {
+    private List<ScoreModel> StudentScoreList1 { get; set; } = new List<ScoreModel>();
+    private List<ScoreModel> StudentScoreList2 { get; set; } = new List<ScoreModel>();
+
+    protected override void OnInitialized()
+    {
+        var thisYear = DateTime.Now.Year;
+        Years = new object[] { thisYear - 4, thisYear - 3, thisYear - 2, thisYear - 1, thisYear };
+
+        var rnd = new Random();
+
+        for (int i = 1; i <= Years.Count(); i++)
+        {
+            int randomValue = rnd.Next(30, 50);
+
+            StudentScoreList1.Add(new ScoreModel()
+            {
+                Year = thisYear - 4 + i,
+                LowScore = randomValue,
+                HighScore = randomValue + 30
+            });
+
+            StudentScoreList2.Add(new ScoreModel()
+            {
+                Year = thisYear - 4 + i,
+                LowScore = randomValue + 10,
+                HighScore = randomValue + 40
+            });
+        }
+
+        base.OnInitialized();
+    }
+
+    public class ScoreModel
+    {
+        public int Year { get; set; }
+        public int LowScore { get; set; }
+        public int HighScore { get; set; }
+    }
+}
+````
+
+
+## Column Chart Specific Appearance Settings
+
+### Labels
+
+Each data item is decorated with a text label. You can control and customize them through the `<ChartCategoryAxisLabels />` and its children tags.
+
+* `Visible` - hide all labels by setting this parameter to `false`.
+* `Step` - renders every n-th labels, where n is the value(double number) passed to the parameter.
+* `Skip` - skips the first n labels, where n is the value (double number) passed to the parameter.
+* `Angle` - rotates the labels with the desired angle by n degrees, where n is the value passed to the parameter. It can take positive and negative numbers. To set this parameter use the `< ChartCategoryAxisLabelsRotation />` child tag.
```

Comment:
```suggestion
* `Angle`—Rotates the labels with the desired angle by n degrees, where n is the value passed to the parameter. It can take positive and negative numbers. To set this parameter, use the `< ChartCategoryAxisLabelsRotation />` child tag.
```